ebpfapi/dllmain.cpp

```diff
@@ -21,8 +21,10 @@ DllMain(HMODULE hModule, unsigned long ul_reason_for_call, void* lpReserved)
         }
         break;
     case DLL_THREAD_ATTACH:
+        ebpf_api_thread_local_initialize();
         break;
     case DLL_THREAD_DETACH:
+        ebpf_api_thread_local_cleanup();
```

Comment:
Is it possible that DLL_PROCESS_DETACH gets trigged but not DLL_THREAD_DETACH? In other words, do we ever care about invoking ebpf_api_thread_local_cleanup in DLL_THREAD_DETACH?

Reply:
DLL_PROCESS_DETACH means the process is being terminated. All resources associated with the process will be terminated, so cleaning up handles is moot.